lumicks/pylake/tests/test_file/test_file.py

```diff
@@ -461,3 +461,12 @@ def cropped_h5(name, crop=None):
                 "PointScan1" not in f["Point Scan"]
                 or not f["Point Scan"]["PointScan1"].red_photon_count
             )
+
+
+def test_detector_mapping(h5_custom_detectors):
+    f = pylake.File.from_h5py(
+        h5_custom_detectors,
+        rgb_to_detectors={"Red": "Detector 1", "Green": "Detector 2", "Blue": "Detector 3"},
+    )
```

Comment:
```suggestion
    )
    assert np.any(f.red_photon_count.data)
```
Maybe verify that we actually have the data present (I could still remove the mapping entirely without triggering an error).